<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-69OQC4Q" ignored="2" name="Testng(failed)" tests="2" failures="1" timestamp="2022-02-25T07:24:47 IST" time="0.238" errors="0">
  <testcase name="m3" time="0.042" classname="com.aspire.framework.testNG.FailedXmlCreation"/>
  <testcase name="m1" time="0.01" classname="com.aspire.framework.testNG.FailedXmlCreation">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at com.aspire.framework.testNG.FailedXmlCreation.m1(FailedXmlCreation.java:15)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- m1 -->
  <testcase name="m4" time="0.0" classname="com.aspire.framework.testNG.FailedXmlCreation">
    <ignored/>
  </testcase> <!-- m4 -->
  <testcase name="m2" time="0.0" classname="com.aspire.framework.testNG.FailedXmlCreation">
    <ignored/>
  </testcase> <!-- m2 -->
</testsuite> <!-- Testng(failed) -->
